ui/src/utils/PromptApi.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: merge with UtteranceApiService.js (in Ashley's PR)
+
+class PromptApi {
+    constructor() {}
+
+    getNewPrompt() {
+        return fetch('/prompt')
+            .then((resp) => resp.json())
+            .then((prompt) => {
+                if (Object.keys(prompt).length === 0) {
+                    return {
+                        type: 'EMPTY',
+                        body: '',
+                    };
+                } else {
+                    return prompt;
+                }
+            });
+    }
+
+    resetAllPromptsUnread() {
+        return fetch('/prompt/all', {method: 'POST'}).then((resp) =>
```

Comment:
nit: same as above, no worries if you disagree, though

Reply:
will change; i prefer promise syntax but we should stick with one for consistency and you've already written async/await code